* Change log dir from /var/lib/log to /var/log (#281439, Martin von Gagern)
* Change user and group name from _tor to tor (#281439, Martin von Gagern)
* Avoid error if tor is not running (#178975, Gustavo Felisberto)

References:
http://bugs.gentoo.org/281439
http://bugs.gentoo.org/178975

--- tor-0.2.1.19/contrib/tor.logrotate.in
+++ tor-0.2.1.19/contrib/tor.logrotate.in
@@ -1,4 +1,4 @@
-@LOCALSTATEDIR@/log/tor/*log {
+/var/log/tor/*.log {
 	daily
 	rotate 5
 	compress
@@ -6,9 +6,9 @@
 	missingok
 	notifempty
 	# you may need to change the username/groupname below
-	create 0640 _tor _tor
+	create 0640 tor tor
 	sharedscripts
 	postrotate
-		/etc/init.d/tor reload > /dev/null
+		/etc/init.d/tor reload > /dev/null || true
 	endscript
 }
